setup.py

```diff
@@ -285,6 +285,7 @@ def create_dir_symlink(src, dest):
           'Documentation': 'https://deepspeed.readthedocs.io',
           'Source': 'https://github.com/microsoft/DeepSpeed',
       },
+      dependency_links=['https://developer.intel.com/ipex-whl-stable-cpu'],
```

Comment:
Is this different from https://pypi.org/project/intel-extension-for-pytorch/?

Reply:
Will remove this line.